build/dotnet/dotnet-install.ps1

```diff
@@ -89,25 +93,6 @@ function Say-Invocation($Invocation) {
     Say-Verbose "$command $args"
 }
 
-function Invoke-With-Retry([ScriptBlock]$ScriptBlock, [int]$MaxAttempts = 3, [int]$SecondsBetweenAttempts = 1) {
```

Comment:
I'm fine with deleting this for now.  CLI has agreed to add this to their copy of the scripts: https://github.com/dotnet/cli/issues/6687